runelite-client/src/main/java/net/runelite/client/RuneLite.java

```diff
@@ -77,6 +80,14 @@
 	public static final File RUNELITE_DIR = new File(System.getProperty("user.home"), ".runelite");
 	public static final File PROFILES_DIR = new File(RUNELITE_DIR, "profiles");
 	public static final File SCREENSHOT_DIR = new File(RUNELITE_DIR, "screenshots");
+	private static final File LOGS_DIR = new File(RUNELITE_DIR, "logs");
+	private static final File LOG_FILE = new File(LOGS_DIR, "client.log");
+	private static final RuneLiteProperties PROPERTIES = new RuneLiteProperties();
+	// Launcher was updated to check it's own version starting at v2.
+	// If they are using a version below 2 (Launcher 1.X.X) the client needs to force them to dl the updated launcher
+	private static final Pattern OUTDATED_LAUNCHER_VERSION = Pattern.compile("Launcher 1\\..*");
```

Comment:
rather than having to do this with regex, which is difficult, it would probable be easier to just use strcmp with the last good version, and hope we don't need to release a 2 digit version number